snp_application/src/snp_widget.cpp

```diff
@@ -122,6 +122,12 @@ SNPWidget::SNPWidget(rclcpp::Node::SharedPtr rviz_node, QWidget* parent)
   bt_node_->declare_parameter<float>(IR_RGBD_DEPTH_SCALE_PARAM, 1000.0);
   bt_node_->declare_parameter<float>(IR_RGBD_DEPTH_TRUNC_PARAM, 1.1f);
   bt_node_->declare_parameter<bool>(IR_LIVE_PARAM, true);
+  bt_node_->declare_parameter<int>(IR_MIN_FACES_PARAM, std::numeric_limits<int>::max());
```

Comment:
I feel like this would make more sense to default to 0

Reply:
Good catch; that was my intent but I got it backwards

Reply:
That would've made for very painful debugging if left unset.